app/src/components/fields/fieldErrorHint/fieldErrorHint.jsx

```diff
@@ -232,6 +232,10 @@ const messages = defineMessages({
     id: 'MembersPageToolbar.membersSearchHint',
     defaultMessage: 'Member name must not be empty',
   },
+  descriptionHint: {
+    id: 'EditItemModal.descriptionHint',
+    defaultMessage: "Description should have size from '0' to '2048' symbols",
+  },
```

Comment:
As we use the same modal for launch, suite and so on, I've asked to use one message for all levels

Reply:
As I know we have selector to detect level
Also in case of any changes of the task you should ask to update it to avoid misunderstanding between owner and reviewers, qa team